dylint/src/driver_builder.rs

```diff
@@ -87,7 +90,7 @@ fn dylint_drivers() -> Result<PathBuf> {
         ensure!(dylint_drivers.is_dir());
         Ok(dylint_drivers.to_path_buf())
     } else {
-        let home = var(env::HOME)?;
+        let home = dirs::home_dir().ok_or_else(|| anyhow!("Could not find the HOME Directory."))?;
```

Comment:
```suggestion
        let home = dirs::home_dir().ok_or_else(|| anyhow!("Could not find HOME directory"))?;
```
Nit to be consistent with error messages elsewhere.